rest/members.go

```diff
@@ -13,7 +13,7 @@ func NewMembers(client Client) Members {
 
 type Members interface {
 	GetMember(guildID snowflake.ID, userID snowflake.ID, opts ...RequestOpt) (*discord.Member, error)
-	GetMembers(guildID snowflake.ID, opts ...RequestOpt) ([]discord.Member, error)
+	GetMembers(guildID snowflake.ID, limit int, opts ...RequestOpt) ([]discord.Member, error)
```

Comment:
can you also add the `after` param?

Reply:
i updated this in the most recent commit